src/hotspot/share/gc/shenandoah/shenandoahControlThread.cpp

```diff
@@ -479,7 +479,7 @@ void ShenandoahControlThread::request_gc(GCCause::Cause cause) {
   assert(GCCause::is_user_requested_gc(cause) ||
          GCCause::is_serviceability_requested_gc(cause) ||
          cause == GCCause::_metadata_GC_clear_soft_refs ||
-         cause == GCCause::_codecache_GC_threshold ||
```

Comment:
Please leave `_codecache_GC_threshold` cause here. Otherwise, it causes a few tests fail.